ofctrl/fgraphSwitch.go

```diff
@@ -54,13 +54,6 @@ func (self *OFSwitch) initFgraph() error {
 	normalLookup.outputType = "normal"
 	normalLookup.portNo = openflow13.P_NORMAL
 	self.normalLookup = normalLookup
-
-	// Clear all existing flood lists
-	groupMod := openflow13.NewGroupMod()
```

Comment:
No flooding is not configured by default, so this cleanup operation is meaningless. A separate patch will be used to clean up the other legacy code.